src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -1045,4 +1051,47 @@ public static Font getTooltipFont(@Nullable Font forcedFont, @Nonnull ItemStack
                 .toList();
     }
 
+    public static void createWorldConfirmationScreen(
+            LevelStorageSource save, String worldName, boolean creatingWorld,
+            Function4<LevelStorageSource.LevelStorageAccess, RegistryAccess.RegistryHolder, ResourceManager, DataPackConfig, WorldData> f4,
+            Function<Function4<LevelStorageSource.LevelStorageAccess, RegistryAccess.RegistryHolder, ResourceManager, DataPackConfig, WorldData>, Runnable> runAfter)
+    {
+        Component title = new TranslatableComponent("selectWorld.backupQuestion.experimental");
+        Component msg = new TranslatableComponent("selectWorld.backupWarning.experimental");
```

Comment:
I wonder if the second title (`msg`) should be a Forge-provided screen which notes that proceeding will cause the screen to not appear again and the warning icon on the selection screen to appear.